src/Zynga/Framework/PgData/V1/Testing/BasePgDataTest.hh

```diff
@@ -0,0 +1,96 @@
+<?hh //strict
+
+namespace Zynga\Framework\PgData\V1\Testing;
+
+use Zynga\Framework\Testing\TestCase\V2\Base as TestCase;
+
+use Zynga\Framework\Cache\V2\Driver\InMemory;
+use Zynga\Framework\Cache\V2\Factory as CacheFactory;
+use Zynga\Framework\Database\V2\Factory as DatabaseFactory;
+use Zynga\Framework\Database\V2\Driver\Mock as MockDriver;
+use Zynga\Framework\Lockable\Cache\V1\Factory as LockableCacheFactory;
+use Zynga\Framework\ShardedDatabase\V3\Driver\Mock as MockShardedDriver;
+use Zynga\Framework\ShardedDatabase\V3\Factory as ShardedDatabaseFactory;
+
+abstract class BasePgDataTest extends TestCase {
```

Comment:
i am guessing this is expected to be extended for all pgdata tests?

Reply:
That is correct, however I forgot to add class documentation for this class to explain it.

Reply:
Makes sense, lets add a class doc here